.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "2.0.0",
+    "tasks": [{
+        "label": "compile-yul-preprocess",
+        "command": "cd etc/system-contracts && yarn preprocess && yarn hardhat run ./scripts/compile-yul.ts && cd ../..",
```

Comment:
we should use 'yarn build-bootloader' (which is doing the compile-yul.script')

also you can pass --cwd to yarn - to avoid doing 'cd' calls.

Reply:
Good catch. Done.